locales/en/app.ftl

```diff
@@ -10,6 +10,10 @@
 -brand-Mozilla = Mozilla
 -brand-HIBP = Have I Been Pwned
 -brand-fxa = Firefox Account
+-brand-pocket = Pocket
+-brand-lockwise = Firefox Lockwise
+-brand-firefox-browser = Firefox Browser
```

Comment:
Wait, aren't we allow languages to localize browser? If so, this shouldn't be a brand, and strings should use `{-brand-name} Browser`

Reply:
One more note: we should change the comment above into a section comment (with two hashes, and an empty line after), and reset the section comment at the end of the brands block with just `##`.

Reply:
That's unfortunate. At this point, we should expose the entire "brand", otherwise it's a mess to localize piece by piece.

```
# "Firefox Browser" needs to remain in English, "for Desktop" can be localized
 -brand-firefox-browser-desktop = Firefox Browser for Desktop

# "Firefox Browser" needs to remain in English, "for Mobile" can be localized
 -brand-firefox-browser-mobile = Firefox Browser for Mobile
```

To clarify one point: the reason to have terms assigned to brands, is that it makes a lot easier to check what happens across languages.
- A message missing a reference brand will raise a warning in compare-locales
- You only need to look at that string, not all strings using the brand

Reply:
Let's hold on a bit more, the ongoing conversations seem to force us to use the current approach instead (keep "Firefox Browser" as a brand).